### PR TITLE
Raise test coverage to ≥90% (Aone #84428597)

### PR DESCRIPTION
**Pull request by yndu13**
## Summary
- Add unit tests covering DefaultLogger/ClientLogger overloads, LogLevel null, Validate/IOUtils/HostnameVerifier success paths, and Tea helpers (null interceptor chain, toWriteable, interrupted sleep).
- Cover OkHttp CA trust-store setup with bundled test PEMs (no CI CA secret), proxy authenticator callback, and proxy exception paths.
- Extend TeaModel confirmType/parseObject, TeaResponse read failures, OkRequestBody contentLength, and ClientHelper cache-key fields for keepAlive/maxIdle.

### CODE REVIEW

File: src/test/resources/test-ca-dual.pem [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+sH6We/jov7GbMSj6uUQALuJwAU3Ct1igmUJ1rvvv9zvQmpZkV5JiP+bQPEvrMAra
+QLCamQXhA2oyK2dddsUFEnZ4FunXUSrCODl0LsXdu9FBAgMBAAGjUzBRMB0GA1Ud
+DgQWBBSV7EHluzoN45UEXZaa8y3+6cBeETAfBgNVHSMEGDAWgBSV7EHluzoN45UE
+XZaa8y3+6cBeETAPBgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQA+
+6hX9+1ncRJwR4zz3e/dN3rs8Q6xtFOuft8oXMS8DJgaJpjnnP/bRDg8+qozjtlai
+5b63V9NKr/iaKWLlEC6z+nY7xhAsETyN2By6tVPy1DrKaTdWvUzM3fG5nEquUvmh
+W/IXexXzHPe/L8FfZDD80EoEzM73D5GyhQtX488npULYyyPJTAfR5q4CHzbubq9m
+kfOif6MH4s0sgDZkcPFNt5dTKj7OUIbVljINEcd6VP+I6m1Wb4vYOBttqQ9jLgLm
+8hc09omOX3/qlRvQuQbdGNaAwRLE8DNtdR9rFIG3Gm2MZaVwDwQnRaeF+uKcFL9u
+MRIs49xwswJtg+AFGO6D
```
Comment (yndu13) [inline src/test/resources/test-ca-dual.pem]: 这些证书是否都是假的，如果是真的，不能直接写在代码里。

Comment (yndu13) [reply src/test/resources/test-ca-dual.pem]: 确认：原先 PEM 是 `CN=test.local` 的自签测试夹具，不是生产证书，也未提交私钥。

为消除「真证书入库」疑虑，已删除 `src/test/resources/test-ca*.pem`，改为单测运行时用 `keytool` 生成 ephemeral 自签 CA（`OU=UnitTestFixture`），覆盖路径不变。请 re-review。
